src/frontend/src/pages/NewTransactionPage.vue

```diff
@@ -49,7 +49,11 @@ import Button from "../components/Button.vue";
 })
 export default class NewTransactionPage extends Vue {
   newTransaction() {
-    const values = { amount: 10, contact: "email" };
+    const values = {
+      amount: 1,
+      contact:
+        "-----BEGIN PUBLIC KEY-----\r\nMIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQCnSXKzVWvTV/wSiG9Rimapz0eA\r\nza9atWCXnU/hTXvOdHpxCpejzOUsB5hPbFC7EdUj4ffHnj/IPYIOgMNzX+i8SwHa\r\nz+gl0K7n8JUzufRnom4/WMHMpkvqUYnVT9/TQLOMyBd/uEEh7is7ioX8EOFMevDu\r\n98aZ4ouHDO9YTNlH4QIDAQAB\r\n-----END PUBLIC KEY-----\r\n",
```

Comment:
TODO: Change this from a hardcoded value to accepting the input passed in the input field